resources/views/platform/components/game/related-games-table.blade.php

```diff
@@ -0,0 +1,187 @@
+            function togglePlayListItem(id)
+            {
+                $.post('/request/user-game-list/toggle.php', {
+                    type: 'play',
+                    game: id
+                })
+                .done(function () {
+                    $("#add-to-list-" + id).toggle();
+                    $("#remove-from-list-" + id).toggle();
+                    if ($("#add-to-list-" + id).is(':visible')) {
+                        $("#play-list-button-" + id).prop('title', '{{ $addButtonTooltip }}');
+                    } else {
+                        $("#play-list-button-" + id).prop('title', '{{ $removeButtonTooltip }}');
+                    }
+                });
+            }
+        </script>
+        @endif
+
+        @foreach ($consoles as $console)
+            @if ($filterOptions['console'])
+                <h2 class="flex gap-x-2 items-center text-h3">
+                    <img src="{{ getSystemIconUrl($console->ID) }}" alt="Console icon" width="24" height="24">
+                    <span>{{ $console->Name }}</span>
+                </h2>
+            @endif
+
+            <div><table class='table-highlight mb-4'><tbody>
+
+            <tr>
+                <th style='width:28%'>Title</th>
+                <th style='width:12%; cursor: help' class='text-right'
+                    title='The number of achievements in the set'>Achievements</th>
+                <th style='width:10%; cursor: help' class='text-right'
+                    title='The number of points associated to achievements in the set'>Points</th>
+                <th style='width:10%; cursor: help' class='text-right'
+                    title='As estimate of rarity for achievements in the set'>RetroRatio</th>
+                <th style='width:10%; cursor: help' class='text-right'
+                    title='The number of leaderboards in the set'>Leaderboards</th>
+                <th style='width:8%; cursor: help' class='text-right'
+                    title='The number of users who have played the set'>Players</th>
+                <th style='width:8%; cursor: help' class='text-right'
+                    title='The number of open tickets for achievements in the set'>Tickets</th>
+                @if ($userProgress !== null)
+                    <th style='width:8%; cursor: help' class='text-center'
+                        title='Indicates how close you are to mastering a set'>Progress</th>
+                    <th style='width:6%; cursor: help' class='text-center'
+                        title='Whether or not the game is on your want to play list'>Backlog</th>
+                @endif
+            </tr>
+            <?php $count = $achievementCount = $pointCount = $leaderboardCount = $ticketCount = 0; ?>
+            @foreach ($games as $game)
+                @if ($filterOptions['console'] && $game['ConsoleID'] != $console['ID'])
+                    @continue
+                @endif
+                <tr>
+                    <td>
```

Comment:
The multiline avatars gives the appearance of the rows being fairly "snug".

```suggestion
                    <td class="py-2">
```

Reply:
Done